# Using metatranscriptomics in estimating diversity and composition of zooplankton communities

Mark Louie D. Lopez, Ya-Ying Lin, Mitsuhide Sato, Chih-hao Hsieh, Fuh-Kwo Shiah, and Ryuji J. Machida

Biodiversity Program, Taiwan International Graduate Program, Academia Sinica, Nankang District, Taipei 11529, Taiwan, Republic of China.*

Department of Life Science, National Taiwan Normal University, Wenshan District, Taipei 11677, Taiwan, Republic of China.*

Biodiversity Research Center, Academia Sinica, Nankang District, Taipei 11529, Taiwan, Republic of China*

Department of Environment and Fisheries Resources, Nagasaki University, Nagasaki City 852-8521, Nagasaki Prefecture, Japan.*

Institute of Oceanography, National Taiwan University, Daâ€™an District, Taipei 10617, Taiwan, Republic of China.*

Environmental Change Research Center, Academia Sinica, Nankang District, Taipei 11529, Taiwan, Republic of China*

## Abstract

DNA metabarcoding is a rapid, high-resolution tool used for biomonitoring complex zooplankton communities. However, diversity estimates derived with this approach can be biased by the co-detection of sequences from environmental DNA (eDNA), nuclear-encoded mitochondrial (NUMT) pseudogene contamination, and taxon-specific PCR primer affinity differences. To avoid these methodological uncertainties, we tested the use of metatranscriptomics as an alternative approach for characterizing zooplankton communities. Specifically, we compared metatranscriptomics with PCR-based methods using genomic (gDNA) and complementary DNA (cDNA) amplicons, and morphology-based data for estimating species diversity and composition for both mock communities and field-collected samples. Mock community analyses showed that the use of gDNA mitochondrial cytochrome c oxidase I (mtCO1) amplicons inflates species richness due to the co-detection of extra-organismal eDNA contaminants. Significantly more amplicon sequence variants, nucleotide diversity, and indels were observed with gDNA amplicons than with cDNA, indicating the presence of putative NUMT pseudogenes. Moreover, PCR-based methods failed to detect the most abundant species in mock communities due to priming site mismatch. Overall, metatranscriptomics provided estimates of species richness and composition that closely resembled those derived from morphological data. The use of metatranscriptomics was further tested using field-collected samples, with the results showing consistent species diversity estimates among biological and technical replicates. Additionally, temporal zooplankton species composition changes could be monitored using different mitochondrial markers. These findings demonstrate the advantages of metatranscriptomics as an effective tool for monitoring diversity in zooplankton research.
